lib/content.js

```diff
@@ -5,6 +5,18 @@ var specSchema= require('./spec-schema');
 
 var mediaTypeRe = /^\s*([^;]+)/i;
 
+var ContentType = {
```

Comment:
I like this :+1: 

Could we though move it into a separate `content-type.js` module and export the functions? :thought_balloon: